Rendering/WDE.MapSpawns/Rendering/SpawnGizmo.cs

```diff
@@ -74,7 +74,7 @@ public override Quaternion GetRotation(SpawnInstance item)
         if (item is CreatureSpawnInstance cr)
             return Quaternion.CreateFromAxisAngle(Vectors.Up, cr.Creature!.Orientation);
         else if (item is GameObjectSpawnInstance go)
-            return go.GameObject!.Rotation;
+            return Quaternion.CreateFromAxisAngle(Vectors.Up, go.GameObject!.Orientation);
```

Comment:
gameobjects can be rotated in all axes, why this change?

Reply:
Yeah, this is a weird one and probably needs some testing. When i spawned my first few gameobjects and tried rotating them I didnt get the same results in game as in the editor. Then i found out that gameobject (just like creature) has also orientation and started using that, despite of course losing two axis. But I know what you mean, from a logic stand point this doesnt make sense, but it works. Will test some more and go back to inital implementation once its working how its supposed to.

Reply:
So there is something different in how the GameView renders some things and I think this i due to rotation parameters, but I cant put my finger where and why and I also kinda dont care enough :D this only happens when hand modifying some rotation values in isolation.

Game View Editor:

![image](https://github.com/BAndysc/WoWDatabaseEditor/assets/5873063/f34a38d9-fd18-4b68-b1d8-f31f839d7402)

In game:

![image](https://github.com/BAndysc/WoWDatabaseEditor/assets/5873063/fe9bef81-2638-420c-8e89-2b37dfbc8fc9)

As you can see gameobjects can get stretched/squished, game seems to make sure that no matter the rotational values the gameobject always has the right proportions.

As this has nothing to do with my "fix" I reverted back to your initial implementation